frontend/src/components/events/index.css

```diff
@@ -17,8 +17,8 @@ tr {
 }
 
 .eventImage {
-  max-width: 240px;
-  max-height: 210px;
+  width: 400px;
```

Comment:
agreed this definitely makes the image look better 👍 

But the width now extends outside the card border. Maybe take a look at modifying `.event_container` to fix that?